python/paddle/fluid/framework.py

```diff
@@ -574,6 +574,74 @@ def find_name(var_list, name):
             self.desc.infer_var_type(self.block.desc)
             self.desc.infer_shape(self.block.desc)
 
+    def equal(self, other):
+        for k, v in self.__dict__.iteritems():
+            if isinstance(v, core.OpDesc) or isinstance(
+                    v, Program) or isinstance(v, Block):
+                continue
```

Comment:
can you favor if-else over if-continue? It's hard to know the `if`a you added is logically complete.

Reply:
Done.